OSDOCS-10219: adds multus assembly to MicroShift (#74481)
openshift/openshift-docs: modules/microshift-multus-intro.adoc

// Module included in the following assemblies:
//
// * microshift_networking/microshift-cni-multus.adoc

:_mod-docs-content-type: CONCEPT
[id="microshift-multus-intro_{context}"]
= Additional networks in {microshift-short}

> During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the Multus CNI plugin, you can define _additional networks_ based on available CNI plugins. You can then attach one or more of these networks to your pods. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.

Reviewer: > Using the Multus CNI plugin, you can define _additional networks_ based on available CNI plugins. You can then attach one or more of these networks to your pods

I wouldn't say that using Multus CNI you can define additional networks.
It's true to some extent about `bridge` plugin (because it can create the bridge if it doesn't exist) but other plugins connect to existing networks, like macvlan or ipvlan can be used to connect Pod directly to host's network.
To me, it's more about adding additional interfaces to Pod from (mostly) existing networks

Author: Recast the language.

[id="additional-network-use-cases"]

> == Use case for an additional network

Reviewer 2: I would suggest a heading that describes the use case--something like "Using additional networks for network isolation" or similar.

> You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, if you want pods to access a network on the host and also communicate with devices deployed to the edge that are on an isolated operator network or are periodically disconnected.

Reviewer 2: ```suggestion
You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, you can configure an additional interface if you want pods to access a network on the host and also communicate with devices deployed to the edge that are on an isolated operator network or are periodically disconnected.
```

Reviewer 2: I feel like this is an incomplete sentence and needs a verb, so I added a suggestion; feel free to rewrite if that's not accurate.

Isolating network traffic is useful for the following performance and security reasons:

> Performance:: You can send traffic on two different planes to manage how much traffic is along each plane.

Reviewer 2: ```suggestion
Performance:: You can send traffic on two different planes to manage the amount of traffic on each plane.
```
